### PR TITLE
update _declarative_constructor location

### PR DESCRIPTION
**Pull request by cnlg-lanl**
Updated location of _declarative_constructor now located in sqlalchemy.orm.decl_base.py . Tried to leave exception for sqlalchemy 1.3.4 users

### CODE REVIEW

**Review by jkmacc-LANL — COMMENTED** [review]
This is great!  Could you also please edit the setup.py to include SQLAlchemy `<=1.4`, similar to what was done [here](https://github.com/samuelchodur/pisces/blob/sa-14-support/setup.py#L30)?  There was one other change in that user's diff about the use of `__mapper__`, but maybe we should wait until we see any issues there ourselves.

File: setup.py [hunk 1]
```diff
@@ -27,7 +27,7 @@
       url='https://github.com/LANL-Seismoacoustics/pisces',
       download_url='https://github.com/LANL-Seismoacoustics/pisces/tarball/0.3.2',
       keywords = ['seismology', 'geophysics', 'database'],
-      install_requires=['numpy','obspy>=1.0','sqlalchemy >1.0, <1.4','Click'],
+      install_requires=['numpy','obspy>=1.0','sqlalchemy<=1.4', <1.4','Click'],
```
Comment (jkmacc-LANL) [inline setup.py:30]: Oops, I guess also need to remove the <1.4 here.  Sorry!

Comment (cnlg-lanl) [reply setup.py]: I saw that as soon as I pushed it

**Comment by jkmacc-LANL** [comment]
@cnlg-lanl That external contributor's PR may have used something other than `declarative_base` because of the intent of the SQLA authors.  Again, maybe we can not worry about it for now, since you've found the original function, and we can just see if it creates any problems.

**Comment by jkmacc-LANL** [comment]
I think the other change made by the external user may have been to address the `items()` error we're seeing.  Want me to make it, or are you ok to?

**Comment by cnlg-lanl** [comment]
I'll let you make it and see if that fixes it...doing some research on it as this could be a separate issue. Looking [here](https://github.com/kvesteri/sqlalchemy-continuum/issues/263)

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,13 @@
 
 ## master/main
```
Comment (jkmacc-LANL) [inline CHANGELOG.md:3]: @cnlg-lanl You can make these changes `## 0.4.0` and `## master/main` will be blank, waiting for content from the next PRs.  Thank you!

**Comment by jkmacc-LANL** [comment]
Closes #38 .

**Comment by cnlg-lanl** [comment]
Done

**Comment by jkmacc-LANL** [comment]
Thanks again, @cnlg-lanl.  Huzzah!

**Comment by cnlg-lanl** [comment]
This also takes care of issue #44

**Pull request merged**